pkg/storage/etcd3/watcher.go

```diff
@@ -52,6 +52,7 @@ type watchChan struct {
 	initialRev        int64
 	recursive         bool
 	filter            storage.FilterFunc
+	filterAcceptsAll  bool
```

Comment:
do we need this additional field? can we simply set filter to nil if we accept everything?